src/main/features/agreement/views/lot.njk

```diff
@@ -41,45 +27,38 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full govuk-grid-column-two-thirds-from-desktop">
         <div class="project-overview">
-            <p class="govuk-body">Agreement name: <strong>Digital Specialists and Programmes</strong></p>
-            <p class="govuk-body">Agreement ID: <strong>RM6263</strong></p>
-            <p class="govuk-body">Expiry date: <strong>DD month YYYY</strong></p>
+            <p class="govuk-body">Agreement name: <strong>{{agreement_lot.name}}</strong></p>
+            <p class="govuk-body">Agreement ID: <strong>{{agreement_id}}</strong></p>
+            <p class="govuk-body">Expiry date: <strong>{{agreement_lot.endDate}}</strong></p>
```

Comment:
This is agreement expiry date not the lot expiry date

Reply:
I took it form her
![lotname](https://user-images.githubusercontent.com/91210101/141654975-1ce7dad7-984e-48d7-ae3d-1a183d550011.PNG)

Reply:
You mean, it should be this?
![agreementENDDATE](https://user-images.githubusercontent.com/91210101/141655186-80cccdcd-e520-4736-bbd0-e5d206b46b65.PNG)